public/app/plugins/datasource/graphite/meta.test.ts

```diff
@@ -0,0 +1,40 @@
+import { parseSchemaRetentions } from './meta';
+
+describe('metadata parsing', () => {
+  it('should parse schema retentions', () => {
+    const retentions = '1s:35d:20min:5:1542274085,1min:38d:2h:1:true,10min:120d:6h:1:true,2h:2y:6h:2';
+    const info = parseSchemaRetentions(retentions);
+    expect(info).toMatchInlineSnapshot(`
+      Array [
+        Object {
+          "resolution": "1s",
+          "savedFor": "35d",
+          "window": "20min",
+          "xxx": "5",
+          "yyy": "1542274085",
```

Comment:
@Dieterbe -- what are reasonable names for the internal values?  I am just guessing here :)

"yyyy" seems to either be:  `unixtime`|`true`|`undefined`

Reply:
note we support two different variants with a different amount of fields,
and also the last field can be either a bool or a timestamp.
see https://github.com/grafana/metrictank/blob/master/scripts/config/storage-schemas.conf#L15-L46 for all the details.